wg-data-protection/charter.md

```diff
@@ -0,0 +1,80 @@
+# WG Data Protection Charter
+
+This charter adheres to the [wg-governance] guidance as well as
+the general conventions described in the [Kubernetes Charter README] and
+the Roles and Organization Management outlined in [sig-governance], where
+applicable to a Working Group.
+
+
+## Scope
+
+Data protection in Kubernetes context typically involves backup and recovery
+of two types of entities:
+* Kubernetes API object resources
```

Comment:
The term "data protection" is typically associated with information security and privacy. It sounds like the goals of this working group are not related. Is there a better term we could use to summarize the work here?

Reply:
"Data protection" is typically the term used by backup vendors.  I'll see if I can add some clarification here.

Reply:
It definitely is a matter of perspective. I immediately thought of GDPR, etc... when I read this PR title. [Duckduckgo](https://duckduckgo.com/?q=data+protection) and [Google](https://www.google.com/search?q=data+protection) have top results related to similar topics. If I live in a different bubble and this is the most unambiguous term, feel free to keep it. Just wondering if there's a less overloaded term.

Reply:
I'm reading this article which gives a definition of data protection: https://searchdatabackup.techtarget.com/definition/data-protection.  It says "a large part of a data protection strategy is ensuring that data can be restored quickly after any corruption or loss. Protecting data from compromise and ensuring data privacy are other key components of data protection."  So the first part is what we are focusing on in this WG.  GDPR is important but just not a focus for this WG at this point (I assume backup vendors will have to comply with GDPR in Europe, but they probably don't have to discuss about a solution in this WG).

Reply:
I'm not asking if GDPR will be specifically addressed. I'm pointing out GDPR as a very well known example usage of "data protection"  that is primarily concern is security and privacy. If you do not plan to address these aspects of data protection, then "data loss prevention" or "storage continuity and disaster" recovery would be more accurate and less ambiguous.

Reply:
This WG is covering more than "storage" though as described in the "Scope" section.  Let me think about it and get back to you.

Reply:
Agree that the term can be confused with information security, but as Xing says, it is also a valid term for Backup and Disaster Recovery and is currently used by the vendors that this group intends to attract. It's difficult to come up with another all-encompassing, well known name. We could make it more clear by renaming this to "Data Protection and Disaster Recovery", but it feels a little too long for a WG name. We can also clarify the distinction better in the charter.

Reply:
Added the following to clarify this: 
https://github.com/kubernetes/community/pull/4301/commits/64f4546ce58074f1c1f2a05f9e5a86cde99615df

Reply:
Can you add some prose with links spelling out the definition, I think that would help alot.

Reply:
Added some text to explain the purpose of Data Protection.  Hope that helps.  Thanks.  https://github.com/kubernetes/community/pull/4301/commits/a3d2690f20b3ef745d90939c04816bb212d2e051